jbdevstudio/com.jboss.jbds.central.discovery.earlyaccess/plugin.xml

```diff
@@ -65,6 +65,37 @@ appropriate tests + report issues to plug-in providers as required.</description
          <icon image32="images/arquillian_32.png"/>
          <overview url="http://www.jboss.org/tools"/>
       </connectorDescriptor>
-
+      
+      <!-- Windup -->
+      <connectorDescriptor
+            categoryId="org.jboss.tools.central.discovery.e.migration"
+            certificationId="com.jboss.jbds.discovery.certification.supported.earlyaccess"
+            description="The Windup plug-ins simplify migrating Java applications."
+            groupId="org.jboss.tools.central.discovery.e.migration.windup"
+            id="org.jboss.tools.windup.feature"
+            kind="task"
+            license="EPL (Free)"
+            name="Windup Migration Toolkit"
+            provider="JBoss by Red Hat"
+            siteUrl="${jboss.discovery.earlyaccess.site.url}">
```

Comment:
Pulling the Windup stuff from the Central targetplatform is good from a stability perspective, but if your updates are not aligned to the 3week cadence of jbosstools/devstudio milestone releases, you might want to link to the http://download.jboss.org/jbosstools/neon/development/updates/windup/composite/ site in the plugin.xml files instead.